protovalidate/__init__.py

```diff
@@ -19,8 +19,8 @@
 ValidationError = validator.ValidationError
 Violations = validator.Violations
 
-_validator = Validator()
-validate = _validator.validate
-collect_violations = _validator.collect_violations
+_default_validator = Validator()
+validate = _default_validator.validate
+collect_violations = _default_validator.collect_violations
 
 __all__ = ["CompilationError", "ValidationError", "Validator", "Violations", "collect_violations", "validate"]
```

Comment:
do we need to `from config import Config` and add it to `__all__` here so that consumers can use it?